scripts/lib.star

```diff
@@ -795,7 +821,7 @@ def release_canary_npm_packages_step(edition):
     }
 
 def deploy_to_kubernetes_step(edition, is_downstream=False):
-    if edition != 'enterprise' or not is_downstream:
+    if edition not in ('enterprise', 'enterprise2') or not is_downstream:
```

Comment:
I don't think we want this step for 'enterprise2', whatever the value of 'is_downstream' is.

Reply:
@AgnesToulet Thx, will check.

Reply:
@AgnesToulet Confirmed and fixed.